hkmc2/shared/src/main/scala/hkmc2/bbml/bbML.scala

```diff
@@ -26,7 +26,12 @@ final case class BbCtx(
   env: HashMap[Uid[Symbol], GeneralType]
 ):
   def +=(p: Symbol -> GeneralType): Unit = env += p._1.uid -> p._2
-  def get(sym: Symbol): Option[GeneralType] = env.get(sym.uid) orElse parent.dlof(_.get(sym))(None)
+  def get(sym: Symbol): Option[GeneralType] =
+    if BbCtx.builtinOps(sym.nme) then ctx.get(s"#${sym.nme}") match
```

Comment:
It occurs to me we now have a proper way of retrieving builtin symbols with `getBuiltin` in `Elaborator.Ctx`.

So I think we should rather use that. So the scheme would be: do shadow normal operators like (+) in `bbPrelude.mls`, in elaboration, but on this line, retrieve the correct symbol with `getBuiltin`.

Reply:
The typer can always find the correct symbol if we shadow normal operators in the prelude. The problem is that the JS builder needs to know the symbol should be builtin.